### DISCUSSION TITLE
How to change offset of the region using pyaedt

### DISCUSSION THREAD

**Discussion by aknsj** [ansys/pyaedt#4377] [Q&A] [answered]
Hello!
As shown below, it is possible to modify the offset of each coordinate system of the region in AEDT Icepak . How to achieve this when using pyaedt?
![image](https://github.com/ansys/pyaedt/assets/163245306/755aa871-5a69-4170-badd-8e75b5b53118)
Thank you for advice.

**Comment by Samuelopez-ansys** [comment, marked as answer]
Hi @aknsj ,

Regions are going to be refactored, right @lorenzovecchietti ? The PR is here: https://github.com/ansys/pyaedt/pulls

In the meanwhile, you can create the region:

`region = app.modeler.create_region()`

And change the X padding data like:

```
region_props = region.history().props
region_props["+X Padding Data"] = 100
```

**Reply by aknsj** [reply]
Hi, I tried the above script and it did work, thank you very much for your prompt reply!
